sql-jdbc/src/main/java/org/opensearch/jdbc/types/TimeType.java

```diff
@@ -24,41 +30,56 @@ public Time fromValue(Object value, Map<String, Object> conversionParams) throws
     if (value == null) {
       return null;
     }
+    Calendar calendar = conversionParams != null ? (Calendar) conversionParams.get("calendar") : null;
     if (value instanceof Time) {
```

Comment:
Just curious what these changes in the if-elseif-else below are doing. Is it just for cleaner code, or did this change have to happen for some reason?

Reply:
Here we check what type of argument the method `ResultSet.getTime()` is provided. It can take `java.sql.Time` object, `String` (ex: "1998-08-17 23:59:59, 23:59:59") and `long` (epoch time, for example: 1672270637000). 
In the first `if` it checks the type and returns `java.sql.Time` as no conversions are necessary.
In the following `else if`-s it does conversions accordingly, as the method returns `Time`.